.gitignore

```diff
@@ -32,3 +32,6 @@
 
 # Ignore Spring files.
 /spring/*.pid
+
+# Ignore vi bad write file
+.swp
```

Comment:
This isn't going to ignore anything except files literally named .swp
Use a wildcard: `.swp` -> `*.swp`